internal/pkg/service/stream/storage/statistics/repository/stats_open.go

```diff
@@ -10,6 +13,13 @@ const (
 	sliceOpenKey = "_open"
 )
 
+func (r *Repository) openStatisticsOnSliceOpen() {
+	r.plugins.Collection().OnSliceOpen(func(ctx context.Context, now time.Time, file model.File, slice *model.Slice) error {
+		op.AtomicOpFromCtx(ctx).AddFrom(r.openSlice(slice.SliceKey))
+		return nil
+	})
+}
+
```

Comment:
The `_open` key for slice stats was created by statistics collector, on disk writer open, so there was a certain time when the key did not exist and sometimes it did.

----------

Now the key is created atomically with the creation of the slice.

I need it for the mechanism in the operators, and it is also easier to understand.